### DISCUSSION TITLE
Resizable Chat-Overview Sidebar (Left) for Better Visibility of Long Titles

### DISCUSSION THREAD

**Discussion by phischde** [open-webui/open-webui#7765] [Ideas]
I have noticed that while it is possible to resize the Chat Controls sidebar on the right, the Chat-Overview Sidebar on the left remains fixed in size. This can cause issues for users who need to view the full title of chats, especially in languages like German, where compound nouns or longer titles are common.

I propose adding the ability to resize the left Chat Sidebar to accommodate varying title lengths. This would enhance the usability of the interface and improve accessibility, particularly for users working in languages with naturally longer text.

**Comment by 9000cats** [comment]
I would also like to be able to resize the sidebar!

**Comment by T313C0mun1s7** [comment]
Yes, I too vote that this would very much improve the UI.

**Comment by Classic298** [comment]
already added
